packages/http/src/lib.ts

```diff
@@ -4,7 +4,7 @@ export const $lib = createTypeSpecLibrary({
   name: "@typespec/http",
   diagnostics: {
     "http-verb-duplicate": {
-      severity: "error",
```

Comment:
I don’t feel like I agree with making this a warning, we have the same pattern for other decorators. It doesn’t make sense to specify 2 verb on the exact same node. We should just allow it to override but complain if it’s specified on the same node not operation. Having to suppress that warning when you want to pverrride with op is just feels wrong anyway

Reply:
If we want to remove the warning altogether, I would not be opposed to it.

I don't think I'd bother with checking/warn if you specified it twice on the same node, though - I think you get exactly what you asked for.

Reply:
So i think the following should be an error
```
@get @get op op1(): void;
@get @post op op2(): void;
```

but this should not emit any errors/warning
```
@get op base(): void;

@post op override is base;
```

Reply:
Hhm I guess `duplicate-decorator` which is the common diagnostic we emit for duplicate use is a [warning not error](https://cadlplayground.z22.web.core.windows.net/prs/3637/?c=aW1wb3J0ICJAdHlwZXNwZWMvaHR0cCI7Cgp1c2luZyBUeXBlU3BlYy5IdHRwOwpAc2VydmljZQpuYW1lc3BhY2UgRGVtb1PGFjsKCi8vIFRoaXMgc2hvdWxkIGJlIGFuIGVycm9yCkByb3V0ZSgiMSIpyQwyIikKb3AgZm9vKCk6IHN0cmluZzs%3D&e=%40typespec%2Fopenapi3&options=%7B%22linterRuleSet%22%3A%7B%22extends%22%3A%5B%22%40typespec%2Fhttp%2Fall%22%5D%7D%7D)  so that can stay a warning, still it shouldn't need to be suppress and only emit this diagnositc when annotating the same node twice

Reply:
Is getting warnings for multiple decorator instances on the same node here a blocker? I can add another utility function to handle the case where there can only be one instance in a "family" of decorators on a given node, but it feels like a low priority thing. If I wasn't digging around in this code to begin with, I would have said it goes on the backlog... and I'm tempted to say so even if I'm digging around in this code.

Reply:
Well from experience seeing how a significant amount of azure spec written don’t see those as warning I do think it is quite a bad experience that will lead to more question we have to keep answering